File: aat-lib/src/main/java/ch/bailu/aat_lib/map/layer/gpx/legend/NullLegendWalker.java
package ch.bailu.aat_lib.map.layer.gpx.legend;

import ch.bailu.aat_lib.gpx.GpxPointNode;
import ch.bailu.aat_lib.gpx.GpxSegmentNode;

public final class NullLegendWalker extends LegendWalker {
    @Override
    public boolean doMarker(GpxSegmentNode marker) {
        return false;
    }

    @Override
    public void doPoint(GpxPointNode point) {

    }
}

File: aat-lib/src/main/java/ch/bailu/aat_lib/map/layer/gpx/legend/NullLegendWalker.kt
package ch.bailu.aat_lib.map.layer.gpx.legend

import ch.bailu.aat_lib.gpx.GpxPointNode
import ch.bailu.aat_lib.gpx.GpxSegmentNode

class NullLegendWalker : LegendWalker() {
    override fun doMarker(marker: GpxSegmentNode): Boolean {
        return false
    }

    override fun doPoint(point: GpxPointNode) {}
}
